frontend/public/components/utils/timestamp.tsx

```diff
@@ -30,7 +32,7 @@ const timestampFor = (mdate: Date, now: Date, omitSuffix: boolean) => {
 
   const minuteStr = mdate.getMinutes().toString().padStart(2, '00');
   let timeStr = `${hours}:${minuteStr} ${a}`;
-  if (mdate.getFullYear() !== now.getFullYear()) {
+  if (noRelative || mdate.getFullYear() !== now.getFullYear()) {
```

Comment:
I noticed our `Timestamps` do not display the full year if it is withing the current year:

![image](https://user-images.githubusercontent.com/12733153/62959218-43ca0c80-bdc6-11e9-8606-0e235d41585f.png)

I interpreted `noRelative` to mean 'display full datetimestamp' without any abbreviations, so `noRelative` will always display the year in the date.

Reply:
Maybe we want `noRelative` and something such as `fullDate`?